### DISCUSSION TITLE
Storage 저장관련 오류 해결여부에 관해

### DISCUSSION THREAD

**Discussion by JeongYire** [naverz/zepeto-studio-kor#252] [5. Q&A]
#112 

이 이슈를 다음버전에 해결하시겠다고하였는데
현재 최신 버전인 1.1.2 에서 해결이 되었나요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

내부 사정으로 인해 해당 이슈 해결이 지연되었습니다.

불편을 드려서 죄송합니다.

데이터 저장에는 문제가 없는 상황이라서 우선은 리턴값을 사용하지 않는 방향성으로 진행 부탁드립니다.
